src/api/v1beta1/feature_flags_resources.go

```diff
@@ -31,7 +31,7 @@ func (dk *DynaKube) FeatureStatsdResourcesLimits(resourceName corev1.ResourceNam
 	return statsdResourceRequirements(dk, "limits-"+resourceName)
 }
 
-// E.g. "operator.dynatrace.com/feature-activegate-eec-resources-limits-cpu": "100m"
+// E.g. "feature.dynatrace.com/activegate-eec-resources-limits-cpu": "100m"
```

Comment:
Either remove this comment or elaborate on it, because this function does not return neither the string `"feature.dynatrace.com/activegate-eec-resources-limits-cpu"` nor the map `"feature.dynatrace.com/activegate-eec-resources-limits-cpu": "100m"`

Reply:
done 😉